parser/raylib_api.json

```diff
@@ -2490,11 +2490,20 @@
       "name": "InitWindow",
       "description": "Initialize window and OpenGL context",
       "returnType": "void",
-      "params": {
-        "width": "int",
-        "height": "int",
-        "title": "const char *"
-      }
+      "params": [
+        {
+          "name": "width",
```

Comment:
Maybe we could exchange order of those lines to follow C convention of (`type` `name`).